### PR TITLE
feat: #117 update docs, rename folder pages to logic

### PR DESCRIPTION
**Pull request by Ekaterina1994**

### CODE REVIEW

File: mw-webapp/cypress/screenshots/Button.cy.tsx/An uncaught error was detected outside of a test (failed).png
Comment (Shustovsky) [inline mw-webapp/cypress/screenshots/Button.cy.tsx/An uncaught error was detected outside of a test (failed).png:1]: do we really need this file?

Comment (Ekaterina1994) [reply mw-webapp/cypress/screenshots/Button.cy.tsx/An uncaught error was detected outside of a test (failed).png:1]: Deleted file and added folder to gitignore

**Review by Shustovsky — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -135,11 +135,10 @@ dist
 .vscode
 
 # autosave draw.io
-.$models.drawio.bkp
+*.drawio.*
```
Comment (tritonJS826) [inline .gitignore:138]: Does not look safe

File: mw-webapp/src/logic/wayPage/WayPage.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import {Button} from "src/component/button/Button";
+import {HeadingLevel, Title} from "src/component/title/Title";
+import {DayReportDAL} from "src/dataAccessLogic/DayReportDAL";
+import {ReportsTable} from "src/logic/reportsTable/ReportsTable";
+import styles from "src/logic/wayPage/WayPage.module.scss";
+
+/**
+ * Way page includes {@link ReportsTable}
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/WayPage.tsx:8]: Don't violate encapsulation

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/logic/wayPage/WayPage.tsx (hunk 2 above)
Comment (Ekaterina1994) [reply mw-webapp/src/logic/wayPage/WayPage.tsx:8]: Changed comments

File: .gitignore (hunk 1 above)
Comment (Ekaterina1994) [reply .gitignore:138]: Changed

**Pull request merged**